static-web-apps/tutorial.md

```diff
@@ -12,7 +12,7 @@ title: Objectives & Prerequesites
 
 In this woskhop, you will learn how to create a full application with a `frontend`, a `serverless backend`, `user authentication` and a `database` to store your data.
 
-Don't worry if you are not familiar with Microsoft Azure. This workshop will walk you through some of the most important services if you are a web developer. We will see how to use `Cosmos DB` with the `MongoDB` API, `Azure Static Web Apps` to host your client application and your `Azure Functions` and `GitHub Actions` to automate your app deployment!
+Don't worry if you are not familiar with Microsoft Azure. This workshop will walk you through some of the most important services if you are a web developer. We will see how to use `Cosmos DB` with the `MongoDB` API, `Azure Static Web Apps` to host your client application and `Azure Functions` for your backend and `GitHub Actions` to automate your app deployment!

```

Comment:
Technically, SWA hosts both the frontend and the functions. There is no Azure Function resource created on Azure.

Reply:
oh ok I see :)